setup.py

```diff
@@ -99,6 +95,12 @@ def split_version(version):
 release = 'dev' not in version
 """.lstrip()
 
+# Only import these if the above checks are okay.
+# See https://github.com/astropy/astropy/issues/10429
```

Comment:
Since this is not a temporary workaround, maybe just explain why it has to be here and don't include the URL?

Reply:
Done. Thanks for the review!